browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -2241,26 +2218,11 @@ index 006b4e80c600158637c5d2f148975b758bd0d094..be517dde6a8877ab3525ea24bb76b94d
  
  UseGPUProcessForMediaEnabled:
 diff --git a/Source/WTF/wtf/DateMath.cpp b/Source/WTF/wtf/DateMath.cpp
-index 735467108e3040bb1d23e9ba1a7ab8a2141a8f66..af5bde4f1c547142b2eaf9b17b59d4f30902c0bf 100644
+index 3e7644d1d232f8fc1a6b1572189dd366445a4e70..4234cab369a6d72fbc233c687a54b67221fa54c4 100644
 --- a/Source/WTF/wtf/DateMath.cpp
 +++ b/Source/WTF/wtf/DateMath.cpp
```

Comment:
Now that timezone override logic is available upstream we revert all our changes to Source/WTF/wtf/DateMath.{h,cpp}. I can do this in a follow-up PR if you don't want to do it as part of the roll.